cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/AbstractConductingEquipmentConversion.java

```diff
@@ -254,7 +254,8 @@ public void convertToDanglingLine(int boundarySide, double r, double x, double g
         // In a Dangling Line the CGMES side and the IIDM side may not be the same
         // Dangling lines in IIDM only have one terminal, one side
         addMappingForTopologicalNode(dl, modelSide, 1);
-        context.convertedTerminal(terminalId(modelSide), dl.getTerminal(), 1, powerFlow(modelSide));
+        // We do not have SSH values at the model side, it is a line flow. We take directly SV values
+        context.convertedTerminal(terminalId(modelSide), dl.getTerminal(), 1, powerFlowSV(modelSide));
```

Comment:
What's happen if we don't have any SV profile at the import? Do we really want to keep a dependency on SV profile ? I though we have decided to convert if only if a user parameter is active ? In the config .yml we have
`iidm.import.cgmes.profile-used-for-initial-state-values: SV`

Reply:
Forgot my comment :-)